### DISCUSSION TITLE
Alojamiento Web mas conveniente para todos y para un futuro

### DISCUSSION THREAD

**Discussion by Jimdrer** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#202] [Ideas]
Bien, iniciemos con éste tema.

Lo mas conveniente para nosotros es hacer el alojamiento en GitHub, en lo personal tengo alojado mi portafolio aquí y claro hare lo mismo para toda practica que este haciente durante el curso. 

Les dejo mi portafolio y se pueden basar el ideas si les sirve algo [https://jimdrer.github.io/Portafolio/](url) 

Para hacer esto es muy sencillo de hacer

Una vez estemos dentro de nuestro repo nos dirigimos a la ultima pestaña que dice **Settings** 
![imagen](https://user-images.githubusercontent.com/81352856/156457140-a01a2534-92b9-466a-a1c4-354f00d1a81b.png)

Una vez dentro nos dirigimos a **Pages**
![imagen](https://user-images.githubusercontent.com/81352856/156457500-325b90f6-9590-43f2-aefc-e9d4a1d077ba.png)

Seleccionada la pestaña vamos a seleccionar la rama que contiene nuestro index o Html (automaticamente va a subir la pagina y hacer el hosting) *En mi caso les pongo imagen de mi porta folio porque ya lo tengo alojado*
![imagen](https://user-images.githubusercontent.com/81352856/156458013-c0373978-b8d4-4771-9532-de95fa6eece0.png)

Espero les sea de ayuda o se den a la idea e investiguen.

Saludos a todos Desarrolladores vibra chida

**Comment by JuanSalvador73** [comment]
Muchas gracias por la info. ¿Sabes si se puede poner mas de un sitio en Github, o es limitado como otros hosting gratis? Saludos :D

**Reply by hackit92** [reply]
Si quieres montar algo un poco más en forma yo te recomendaría aprovechar la capa gratuita de AWS o de Oracle, yo tengo un VPS con Oracle y la verdad es una maravilla
